docs/integrations/azure/locations.md

```diff
@@ -0,0 +1,30 @@
+---
+id: locations
+title: Azure DevOps Locations
+sidebar_label: Locations
+description: Documentation on Azure DevOps location integration
```

Comment:
hm i wonder if the wording with "Documentation on" is useful in a documentation portal. This is really really nitpicky but I just got to thinking, should one decide on a certain structure / word form to use for things like this. For example "Registering Azure DevOps locations in the catalog" (-ing form) or whether to be personal or not ("How you can ...") etc

Reply:
Essentially i guess this text should be one that makes sense to show as part of a google (or backstage) search result i guess

Reply:
Agreed, `Documentation on ${title}` essentially feels silly here. I'll make at least the new ones I added a bit more meaningful.